docs/prebuilt_models.rst

```diff
@@ -210,40 +210,40 @@ See the full list of all precompiled model libraries `here <https://github.com/m
 
 .. |
 
-.. .. _prebuilt-models-android:
-
-.. Prebuilt Models on Android
-.. ^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. For more, please see :doc:`the Android page </deploy/android>`.
-
-.. .. collapse:: Click to show details
-
-..   The apk for demo Android app includes the following models. To add more, check out the Android page.
-
-..   .. list-table:: Prebuilt Models for Android
-..     :widths: 15 15 15 15
-..     :header-rows: 1
-
-..     * - Model code
-..       - Model Series
-..       - Quantization Mode
-..       - Hugging Face repo
-..     * - `Llama-2-7b-q4f16_1`
-..       - `Llama <https://ai.meta.com/llama/>`__
-..       - * Weight storage data type: int4
-..         * Running data type: float16
-..         * Symmetric quantization
-..       - `link <https://huggingface.co/mlc-ai/mlc-chat-Llama-2-7b-chat-hf-q4f16_1>`__
-..     * - `RedPajama-INCITE-Chat-3B-v1-q4f16_1`
-..       - `RedPajama <https://www.together.xyz/blog/redpajama>`__
-..       - * Weight storage data type: int4
-..         * Running data type: float16
-..         * Symmetric quantization
-..       - `link <https://huggingface.co/mlc-ai/mlc-chat-RedPajama-INCITE-Chat-3B-v1-q4f16_1>`__
-.. .. for a blank line
-
-.. |
+.. _prebuilt-models-android:
+
+Prebuilt Models on Android
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For more, please see :doc:`the Android page </deploy/android>`.
+
+.. collapse:: Click to show details
+
+  The apk for demo Android app includes the following models. To add more, check out the Android page.
+
+  .. list-table:: Prebuilt Models for Android
+    :widths: 15 15 15 15
+    :header-rows: 1
+
+    * - Model code
+      - Model Series
+      - Quantization Mode
+      - Hugging Face repo
+    * - `Llama-2-7b-q4f16_1`
+      - `Llama <https://ai.meta.com/llama/>`__
+      - * Weight storage data type: int4
+        * Running data type: float16
+        * Symmetric quantization
+      - `link <https://huggingface.co/mlc-ai/mlc-chat-Llama-2-7b-chat-hf-q4f16_1>`__
```

Comment:
This should probably be `https://huggingface.co/mlc-ai/Llama-2-7b-chat-hf-q4f16_1-MLC` instead; and `https://huggingface.co/mlc-ai/RedPajama-INCITE-Chat-3B-v1-q4f16_1-MLC` for RedPajama